cmd/plugin/plugin.go

```diff
@@ -40,17 +40,31 @@ func InstallCommand(out output.Output) *cobra.Command {
 
 			defaultSource := avm.GetDefaultSource()
 
+			if len(args) < 2 {
+				return fmt.Errorf("Need at  least two arguments, plugin name and version, got args: %v", args)
+			}
+
+			pluginName := args[0]
+			pluginVersion := args[1]
+
+			var pluginURL string
+
+			if len(args) > 2 {
+				pluginURL = args[2]
+			}
```

Comment:
Is there a better way in go to do this?

Reply:
we should use flags instead of positional arguments

Reply:
I will have a look right now at flags.

Reply:
I'm using flags in the last commit, not sure if this is the best way to do this.